### PR TITLE
[NUI] Add some APIs into WebView.

### PR DESCRIPTION
**Pull request by huayongxu**
Title, Favicon, ClearAllTilesResources are added.
Meanwhile WebView can be created with arguments argc & args
that will be delivered into web engine.

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by elishateng — APPROVED** [review]
LGTM

**Review by AchoWang — APPROVED** [review]
Looks good to me.

File: src/Tizen.NUI/src/internal/WebView.cs [hunk 1]
```diff
@@ -254,6 +263,17 @@ public WebView(string locale, string timezoneId) : this(Interop.WebView.New2(loc
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Creates an uninitialized WebView.
+        /// <param name="argc">count of args</param>
+        /// <param name="args">args array</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WebView(int argc, string[] args) : this(Interop.WebView.New3(argc, args), true)
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/internal/WebView.cs:272]: I don't think the first argument `argc` is necessary. It can be optained from args.Length.

**Review by rabbitfor — APPROVED** [review]

**Comment by dongsug-song** [comment]
@huayongxu 
Hello, please check the review comments added so far.

and please resolve the conflicts. otherwise please upload new PR and close this PR.
thank you.

**Comment by huayongxu** [comment]
There are some conflicts in this patch.
I will upload a new one.

**Pull request closed without merging**